### PR TITLE
feat: login page

### PR DESCRIPTION
**Pull request by thesudhansupanda**

### CODE REVIEW

File: templates/login.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        <section class="user-info">
+            <div class="user-profile flex align-middle">
+
+                <div class="dropdown">
+                   <div class="avatar flex flex-col-rev">
+                     <button style="color:black;" class="btn">Login</button>
+                   </div>
+                </div>
+
+                <div class="badgewrapper">
```
Comment (shashidwivedi) [inline templates/login.html:31]: You can use a - between badge and wrapper in the class name

File: templates/login.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+        <section class="navlinks">
+            <h2>Paddling</h2>
+            <a href="#" class="nav-items">Home</a>
+            <a href="#" class="nav-items dropdown">Products</a>
+            <a href="#" class="nav-items">OurCause</a>
+        </section>
+
+        <section class="user-info">
+            <div class="user-profile flex align-middle">
```
Comment (shashidwivedi) [inline templates/login.html:23]: Can give more info on flex classname.

File: templates/login.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+                <div class="dropdown">
+                   <div class="avatar flex flex-col-rev">
+                     <button style="color:black;" class="btn">Login</button>
+                   </div>
+                </div>
+
+                <div class="badgewrapper">
+                    <a href="#"><img src="/assets/heart.svg" alt=""></a>
```
Comment (shashidwivedi) [inline templates/login.html:32]: missing alt value

**Review by shashidwivedi — APPROVED** [review]
The rest looks good. Will suggest using prettier. Also see comments

File: templates/login.html (hunk 1 above)
Comment (thesudhansupanda) [reply templates/login.html:31]: alright, we do as a part of refactor

**Pull request merged**